### PR TITLE
Prepare for 4.1.0 release

### LINKED ISSUE #156: Release Regularly
We really need to start releasing more regularly, instead of major releases after we've implemented 20-30 features. There are a lot of bug fixes and enhancements that are being held off because we're waiting to release a big version.

That said, we do want to limit the number of major releases we do, and chunk together as many breaking changes as we can into a single major release. I think we can do this much more effectively if we use a separate branch to hold breaking changes, and only merge breaking changes into that branch until we're ready to release, at which point we would merge the breaking_changes branch into master.  This way we can continue to merge small changes into master and release regularly, but control the release of breaking changes.

Thoughts?

### PR DESCRIPTION
**Pull request by jsmartt**
### Description
Release 4.1.0
Ready after #193, unless someone else has something they need to get in first.

### Issues Resolved
Fixes #156

### Check List
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -434,7 +432,4 @@ First run `$ bundle` (requires the bundler gem), then...
 Note: run `$ rake -T` to get a list of all the available rake tasks.
 
 ## Authors
- - Jared Smartt - [@jsmartt](https://github.com/jsmartt)
- - Henrique Diomede - [@hdiomede](https://github.com/hdiomede)
- - Thiago Miotto - [@tmiotto](https://github.com/tmiotto)
- - Ricardo Piantola - [@piantola](https://github.com/piantola)
+See the [contributors graph](graphs/contributors)
```
Comment (aalexmonteiro) [inline README.md]: Page 404.

**Comment by aalexmonteiro** [comment]
You anticipated me, because I was preparing the release too, but no problem.  :)

Ready to merge. :shipit:

File: README.md (hunk 1 above)
Comment (ricardogpsf) [inline README.md]: It's working well, @aalexmonteiro .
You can test it selecting the release4.1.0 branch, in home url of this repo. See [here](https://github.com/HewlettPackard/oneview-sdk-ruby/tree/release4.1.0)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # Unreleased Changes
+(none)
```
Comment (ricardogpsf) [inline CHANGELOG.md]: Why this section?

Comment (jsmartt) [inline CHANGELOG.md]: Just to keep the header. No other reason.

**Comment by aalexmonteiro** [comment]
Hey guys, we have identified that do not have an endpoint to be implemented in the Artifact Bundle. Ricardo is working on it and is soon ready to be released.

**Comment by aalexmonteiro** [comment]
Hey guys, we have identified that an endpoint has not been implemented implemented, @ricardogpsf is working on it and soon is it will be ready.

**Comment by fgbulsoni** [comment]
> Hey guys, we have identified that an endpoint has not been implemented, @ricardogpsf is working on it and soon is it will be ready.

PR approved for merge, I guess once these two are in we can officially make the release.
@jsmartt would you do the honors?

**Comment by jsmartt** [comment]
Sure thing!

**Comment by aalexmonteiro** [comment]
PR #197 was merged. Ready for release.

**Comment by ricardogpsf** [comment]
:shipit:

**Pull request merged**